### PR TITLE
Intro jupyter sp

### PR DESCRIPTION
**Pull request by Sipa91**
- added Intro_to_Jupyter_notebooks - Notebook from minibootcamp and replaced existing one
- added folder with images
- adapted superstore 5-step analysis-notebook

### CODE REVIEW

File: 3_Analytics_workflow.ipynb [hunk 1]
```diff
@@ -4,16 +4,36 @@
    "cell_type": "markdown",
```
Comment (marmorlade) [inline 3_Analytics_workflow.ipynb:4]: <p>How does an order look like? </p><p>-&gt; What does an order look like? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/pg_workshop/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (marmorlade) [inline 3_Analytics_workflow.ipynb:4]: <p>Count the number of orders and products: use '.nunique' on the order_id and product_id columns to get the number of unique values </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/pg_workshop/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

File: 3_Analytics_workflow.ipynb [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+   "source": [
+    "# 1 Ask questions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Start your Analytics workflow with asking the right/relevant questions.  \n",
+    "It is important to have questions/hypotheses in mind, which lead your analysis - without them you might easily get lost.\n",
```
Comment (philippwendt) [inline 3_Analytics_workflow.ipynb:43]: Maybe rephrase to emphasize how important the Why? is. Sth like: Before analysing data, you should make sure you know why you are doing what you are doing. Have a clear definition of the question you want to answer or what the purpose of the output is you want to generate. Not doing so can easily lead to you wasting your time on unnecessary tasks or getting lost... (in the jungle of insights? lol)

File: 3_Analytics_workflow.ipynb [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+    "  \n",
+    "**Which category of your products is the least successful one?**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2 Load your data\n",
+    "Having your question(s) in mind, it is now time to start your analysis. First, you need to load the data, you want to look at. In Python, there is a specific Python library called *Pandas* which is aimed at working with dataframes. Pandas comes with functions which are able to import your data into a Dataframe format. Thus, as first step, you need to import the Pandas library and read in your data file."
```
Comment (philippwendt) [inline 3_Analytics_workflow.ipynb:54]: "Working with dataframes" - have we told them about dataframes already? If not then I would add a small sentence like: A dataframe is data structure that organizes data into 2-dimensional table of rows and columns.

Comment (Sipa91) [reply 3_Analytics_workflow.ipynb:54]: @marmorlade : You will introduce data frames in your notebook, right?
